.travis/script.sh

```diff
@@ -0,0 +1,28 @@
+#!/usr/bin/env bash
+
+if [ "$TYPE" = 'style' ]; then
+
+	# shellcheck disable=SC2046
+	shellcheck $(find -O3 "$TRAVIS_BUILD_DIR/.travis/" "$TRAVIS_BUILD_DIR/cmake/" -type f -name '*.sh')
+
+else
+
+	mkdir build && cd build || exit
```

Comment:
`mkdir build && cd build` was here before in the `.yml`. I added the exit but with `set -e` it's useless now.